slides/developer-guide/working-with-notes-slides/extract-and-download-a-noteslide-from-a-powerpoint-presentation/_index.md

```diff
@@ -1,134 +1,134 @@
-### **SDK Examples**
-{{< tabs tabTotal="9" tabID="5" tabName1="C#" tabName2="Java" tabName3="PHP" tabName4="Ruby" tabName5="Python" tabName6="Node.js" tabName7="Android" tabName8="C++" tabName9="Perl" >}}
-
-{{< tab tabNum="1" >}}
-
-{{< gist "" "a41a3c7c75241129b94faf7179d42527" "GetNotesSlides.cs" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="2" >}}
-
-{{< gist "" "17b08f624ccca40e351e7204e318237e" "GetNotesSlides.java" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="3" >}}
-
-```php
-    // For complete examples and data files, please go to https://github.com/aspose-slides-cloud/aspose-slides-cloud-php
-
-    use Aspose\Slides\Cloud\Sdk\Api\SlidesApi;
-    use Aspose\Slides\Cloud\Sdk\Api\Configuration;
-    use Aspose\Slides\Cloud\Sdk\Model;
-    use Aspose\Slides\Cloud\Sdk\Model\Requests;
-
-    try {
-        // Create SlidesApi instance
-        $config = new Configuration();
-        $config->setAppSid("MyClientId");
-        $config->setAppKey("MyClientSecret");
-        $slidesApi = new SlidesApi(null, $config);
-
-        $fileName = "test-unprotected.ppt";
-        $index = 2;
-
-        // Upload original document to storage
-        $fileStream = fopen(realpath(__DIR__ . '/../..') . '\resources\\' . $fileName, 'r');
-        $slidesApi->uploadFile($fileName,  $fileStream);
-
-        $request = new Requests\GetNotesSlideRequest($fileName, $index);
-        $result = $slidesApi->getNotesSlide($request);
-        print_r($result);
-
-    } catch (Exception $e) {
-        echo "Something went wrong: ", $e->getMessage(), "\n";
-    }
-```
-
-{{< /tab >}}
-
-{{< tab tabNum="4" >}}
-
-{{< gist "" "2cc36b05065a88cb0737424e4f38f68e" "GetNotesSlides.rb" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="5" >}}
-
-{{< gist "" "88b9472c3f741eae6c606abdd003c791" "GetNotesSlides.py" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="6" >}}
-
-{{< gist "" "bc650902bdc45144b1727d329023dcba" "GetNotesSlides.js" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="7" >}}
-
-{{< gist "" "2b52dabd204b301389d1f4234e9bb0d5" "GetNotesSlides.java" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="8" >}}
-
-{{< gist "" "a41a3c7c75241129b94faf7179d42527" "Coming_Soon.txt" >}}
-
-{{< /tab >}}
-
-{{< tab tabNum="9" >}}
-
-{{< gist "" "a41a3c7c75241129b94faf7179d42527" "Coming_Soon.txt" >}}
-
-{{< /tab >}}
-
-{{< /tabs >}}
-
-### **SDK Source**
-The Aspose Cloud SDK's can be downloaded from the following page: [Available SDK's](/slides/available-sdks/)
+---
+title: "Extract and Download a NoteSlide from a PowerPoint Presentation"
+type: docs
+url: /extract-and-download-a-noteslide-from-a-powerpoint-presentation/
+weight: 30
+---
+
+## **Introduction**
+Aspose.Slides Cloud API allows you to export and download a NoteSlide to a desired format
```

Comment:
Add a final dot

Reply:
This article was only moved and it will be done as a part of the task SLIDESCLOUDDOC-76 (Fix and update "Extract and Download a NoteSlide from a PowerPoint Presentation" article).